runtime/amplitude/src/lib.rs

```diff
@@ -1201,6 +1203,35 @@ impl clients_info::Config for Runtime {
 	type MaxUriLength = ConstU32<255>;
 }
 
+parameter_types! {
+	pub const DecayRate: Perquintill = Perquintill::from_percent(5);
+	pub const MaxCurrencies: u32 = 10;
+}
+
+impl reward_distribution::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type WeightInfo = reward_distribution::SubstrateWeight<Runtime>;
+	type Balance = Balance;
+	type DecayInterval = ConstU32<100>;
```

Comment:
The decay interval should be set to 1 month, according to https://github.com/pendulum-chain/tasks/issues/47
Since we can only measure it in blocks, we have to assume that a decay interval of `216_000` blocks roughly relates to a month, assuming a month consists of 30 days.